### PR TITLE
RHBPMS-4514 - Null Pointer exception occurs for Business Process with…

### PR DESCRIPTION
**Pull request by tsurdilo**
… empty Embedded SubProcess (#750)

### CODE REVIEW

File: jbpm-simulation/src/main/java/org/jbpm/simulation/PathContext.java [hunk 1]
```diff
@@ -15,10 +15,7 @@
 
 package org.jbpm.simulation;
 
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```
Comment (psiroky) [inline jbpm-simulation/src/main/java/org/jbpm/simulation/PathContext.java]: I believe we should not be using star imports.

**Comment by tsurdilo** [comment]
@psiroky sorry will update. ide change it.

**Comment by tsurdilo** [comment]
@psiroky amended

File: jbpm-simulation/src/main/java/org/jbpm/simulation/PathContext.java [hunk 2]
```diff
@@ -15,14 +15,15 @@
 
 package org.jbpm.simulation;
 
+
+import org.eclipse.bpmn2.FlowElement;
+import org.eclipse.bpmn2.SequenceFlow;
```
Comment (psiroky) [inline jbpm-simulation/src/main/java/org/jbpm/simulation/PathContext.java]: Nit-picking, but this is still wrong. This does not follow the `java.*` + `javax.*` first (which is what we should use), nor the alphabetical order (which may be colliding with what we use).

**Comment by tsurdilo** [comment]
@psiroky ok. amended too.

**Comment by tsurdilo** [comment]
jenkins retest this

**Pull request merged**